app/services/AdditionalInformationService.scala

```diff
@@ -38,14 +39,22 @@ class AdditionalInformationService @Inject() (referenceDataConnector: ReferenceD
         additionalInformationList =>
           val isConsignmentCountryOfDestinationInCL009 = userAnswers.get(ItemsDestinationCountryInCL009Page).contains(true)
 
+          val isConsigneeDetails: Boolean =
```

Comment:
Hello @jamie2210, we have an existing approach where we reference values from other services in the pages/external and section/external folders in this service. You might want to migrate this implementation to use the same approach. Also, I reckon just checking that the Consignee section is defined should suffice rather than drilling all the way to the name value.